model/bin/link.tmpl

```diff
@@ -116,7 +116,7 @@
     case $WWATCH3_NETCDF in
       NC3) libs="$libs -L$NETCDF_LIBDIR -lnetcdf" ;;
       NC4) if [ "$mpi_mod" = 'no' ]; then comp="`$NETCDF_CONFIG --fc`"; fi
-           libs="$libs `$NETCDF_CONFIG --flibs`" ;;
+           libs="$libs `$NETCDF_CONFIG --flibs` `$NETCDF_CONFIG --libs`" ;;
```

Comment:
If the netcdf-fortran library is correctly compiled, all the needed dependencies are defined in --flibs.
the option --libs does not exist for all netcdf libraries (like version 4.5.2) so it crashes.